cli/daemon/run.go

```diff
@@ -236,38 +236,54 @@ func (s *Server) Test(req *daemonpb.TestRequest, stream daemonpb.Daemon_TestServ
 	}
 	s.cacheAppRoot(man.AppID, req.AppRoot)
 
-	setupCtx, setupCancel := context.WithTimeout(context.Background(), 30*time.Second)
+	var usesSQLDB bool
+	for _, svc := range parse.Meta.Svcs {
```

Comment:
This can be replaced with `usesSQLDB := requiresSQLDB(parse.Meta)`

Reply:
Missed it! Thanks. It's replaced now